### PR TITLE
Redux installed and added reducer files

### PR DESCRIPTION
**Pull request by haliljon**
## In this PR I was able to:

> - understand the concept of state management.
> - use store, actions, and reducers in React.

### CODE REVIEW

File: src/redux/book/book.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+      ];
+    },
+    removeBook(state, id) {
+      return [...state.filter((book) => book.id !== id)];
+    },
+  },
+});
+
+export const bookActions = bookSlice.actions;
+export default bookSlice.reducer;
```
Comment (jerryowusu) [inline src/redux/book/book.js]: Kindly define ADD_BOOK and REMOVE_BOOK as action types in conjunction with the project requirements. The value of each type needs to follow the [duck pattern](https://github.com/erikras/ducks-modular-redux), so it should be something like: 
`const ADD_BOOK = bookstore/books/ADD_BOOK` and `const REMOVE_BOOK = bookstore/books/REMOVE_BOOK`

File: src/redux/categories/categories.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+      return [
+        {
+          ...state,
+          isChecking: true,
+          message: 'Under construction',
+        },
+      ];
+    },
+  },
+});
```
Comment (jerryowusu) [inline src/redux/categories/categories.js]: Kindly define  CHECK_STATUS as action types in conjunction with the project requirements. The value needs to follow the [duck pattern](https://github.com/erikras/ducks-modular-redux), so it should be something like:
`const CHECK_STATUS = bookstore/categories/CHECK_STATUS` :wink:

**Review by jerryowusu — CHANGES_REQUESTED** [review]
Hi @haliljon  

Good job so far!
Your project is looking fantastic so far :star_struck: 

However, there are few changes you need to make in order to move on to the next level :sunglasses:

almost-there-reach


## Highlights
No linter errors :heavy_check_mark: 
Good job so far :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/categories/categories.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+const categoriesSlice = (state = initialCategoriesState, action) => {
+  switch (action.type) {
+    case CHECK_STATUS:
+      return { ...state, isChecking: true, message: 'Under construction' };
+    default:
+      return state;
+  }
+};
+
+export default categoriesSlice.reducer;
```
Comment (thecodechaser) [inline src/redux/categories/categories.js]: - [ ] Great job implementing the changes requested by the previous reviewer, But you missed updating the reducer export. Please fix the categories reducer export as it should be `export default categoriesSlice;`.

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @haliljon👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by prolajumokeoni — APPROVED** [review]
Hi @haliljon 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- No linter errors ✅ 
- Git flow was used correctly 🚀 
- Good Pr title and description 💪🏿 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**